airbyte-integrations/connectors/source-db2/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/Db2SourceDatatypeTest.java

```diff
@@ -43,7 +43,6 @@ protected JsonNode getConfig() throws Exception {
 
   @Override
   protected void tearDown(final TestDestinationEnv testEnv) {
-    dslContext.close();
```

Comment:
What did this do before?

Reply:
I'm not sure, however, following a `jooq` library version upgrade, `DSLContext` simply no longer implements `AutoClosable`. At the time of the dependency cleanup, I looked into whether there was some other resource release mechanism that I was supposed to use but didn't find any.